### ISSUE TITLE
[unity] NullReferenceException when SkeletonGraphic SkeletonDataAsset is null

### ISSUE THREAD

**Issue by bps86** [EsotericSoftware/spine-runtimes#2294]
Here is the following error

```
NullReferenceException: Object reference not set to an instance of an object
Spine.Unity.MeshGenerator.GenerateSingleSubmeshInstruction (Spine.Unity.SkeletonRendererInstruction instructionOutput, Spine.Skeleton skeleton, UnityEngine.Material material) (at Assets/Spine/Runtime/spine-unity/Mesh Generation/MeshGenerator.cs:153)
Spine.Unity.SkeletonGraphic.PrepareInstructionsAndRenderers (System.Boolean isInRebuild) (at Assets/Spine/Runtime/spine-unity/Components/SkeletonGraphic.cs:700)
Spine.Unity.SkeletonGraphic.LateUpdate () (at Assets/Spine/Runtime/spine-unity/Components/SkeletonGraphic.cs:409)
```

Previously on Spine 3.8, if skeletondataasset was null, it will not give an error during active and inactive playmode. Now in Spine 4.1, when the skeletondataasset is null, it will give this error both during playmode is active and inactive. Was it supossed to be like this or was this a bug?
![Screenshot 2023-05-26_16:34:44_pm](https://github.com/EsotericSoftware/spine-runtimes/assets/59677031/a5d5b9e2-f193-441a-88f5-9cb28aeff956)

**Comment by HaraldCsaszar** [comment]
@bps86 Thanks for reporting! We have fixed this issue on both 4.1 and 4.2-beta branches. New unitypackages are available for download here as usual:
https://esotericsoftware.com/spine-unity-download

**Comment by bps86** [comment]
@HaraldCsaszar  Understood, thanks for the fixes and information.
